facia-press/conf/routes

```diff
@@ -2,4 +2,7 @@ GET        /_healthcheck        conf.HealthCheck.healthcheck()
 GET        /                    controllers.Application.index
 GET        /debug/config        controllers.Application.showCurrentConfig
 
-GET        /pressed/live/*id      controllers.Application.generateLivePressedFor(id)
+GET        /pressed/live/*id    controllers.Application.generateLivePressedFor(id)
+
+GET        /press/live/*path    controllers.Application.pressLiveForPath(path)
+GET        /press/draft/*path   controllers.Application.pressDraftForPath(path)
```

Comment:
Feels wrong for GET to have a side effect. But I guess it's acceptable as a switchable-off emergency URL that needs to be easy to hit...